src/test/java/com/google/sps/AvailabilityTest.java

```diff
@@ -49,20 +52,35 @@
                                                         .setCalendarType("iso8601")
                                                         .setDate(2020, 7, 10)
                                                         .build();
+    
+    private final LocalServiceTestHelper helper =  new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+    
     private AvailabilityServlet servlet;
 
     @Before
-    public void setUp() {		        
-        servlet = new AvailabilityServlet(true);
-        TutorSession.resetIds();
+    public void setUp() {
+        helper.setUp();	        
+
+        servlet = new AvailabilityServlet();
+        servlet.init();
+
+        SampleData sample  = new SampleData();
+        sample.addTutorsToDatastore();
```

Comment:
When this is added the changes from previous tests are not carried over, are they? Is the sample data being added the its initial version?

Reply:
They are not carried over. Since the local datastore helper keeps all the data in memory, the memory is wiped after the end of each test: https://cloud.google.com/appengine/docs/standard/java/tools/localunittesting#datastore-memcache

Yes, the sample data being added is the initial version (unaltered).

Reply:
Perfect!